### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by stefonarch**

### CODE REVIEW

**Comment by stefonarch** [comment]
Any objections?

**Comment by tsujan** [comment]
None from me if you know that it works (I don't use portals).

**Comment by q234rty** [comment]
TL;DR:
We want
```shell
hash dbus-update-activation-environment 2>/dev/null && \
     dbus-update-activation-environment --systemd XDG_CURRENT_DESKTOP=LXQt
```
in startlxqt.

I don't think writing a config file into `~/.config/xdg-desktop-portal/lxqt-portals.conf` currently works under a lxqt session as started in https://github.com/lxqt/lxqt-session/blob/master/startlxqt.in

The issue is, xdg-desktop-portal does not "know" it's a lxqt session, so it won't read `lxqt-portals.conf`. The portal relies on the env var `XDG_CURRENT_DESKTOP` for that, and while the startlxqt script exports said env var, xdg-desktop-portal is usually started via dbus activation, which does not use the same environment.

The command mentioned above tries to update the dbus activation environment as well.

As a workaround before said change is released, one can use `~/.config/xdg-desktop-portal/portals.conf` instead, as x-d-p reads that when `XDG_CURRENT_DESKTOP` is not set.
